docker-compose.yml

```diff
@@ -97,10 +97,14 @@ services:
       context: .
     volumes:
       - "./alws:/code/alws"
+      - "./tests:/code/tests"
```

Comment:
not sure that task_queue really needs volume with tests

Reply:
I just mount that we copy into an image.